data/json/items/comestibles/seed.json

```diff
@@ -197,14 +197,30 @@
     "id": "seed_garlic",
     "copy-from": "seed",
     "price": 50,
+    "name": "garlic seeds",
+    "name_plural": "garlic seeds",
+    "description": "Some onion seeds.",
+    "seed_data": { "plant_name": "garlic", "fruit": "garlic", "byproducts": [ "withered" ], "grow": 65 }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "garlic_clove",
     "name": "garlic clove",
     "name_plural": "garlic cloves",
-    "description": "Cloves of garlic.  Useful as a seasoning, or for planting.",
+    "weight": 3,
+    "color": "white",
+    "spoils_in": 8640,
+    "comestible_type": "FOOD",
+    "symbol": "%",
     "quench": -3,
+    "healthy": 1,
     "nutrition": 3,
-    "weight": 3,
+    "description": "Cloves of garlic.  Useful as a seasoning.",
+    "price": 50,
```

Comment:
If we can still plant garlic seeds, should you leave in the "or for planting" part of the description?